<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_athete_to_team</class>
 <widget class="QDialog" name="add_athete_to_team">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add athlete</string>
  </property>
  <widget class="QLabel" name="name_lable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nmae of Athlete:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="athlete_name_input">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="information_lable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter thr name of the athlete you want to add to team</string>
   </property>
  </widget>
  <widget class="QLabel" name="team_name_lable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Team name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="team_name_input">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="team_mode_input">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Football</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PingPong</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Badminton</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
